nixos/modules/services/web-apps/pingvin-share.md

```diff
@@ -0,0 +1,86 @@
+# Pingvin Share {#module-services-pingvin-share}
+
+A self-hosted file sharing platform and an alternative for WeTransfer.
+
+## Configuration {#module-services-pingvin-share-basic-usage}
+
+By default, the module will execute Pingvin Share backend and frontend on the ports 8080 and 3000.
+
+I will run two systemd services named `pingvin-share-backend` and `pingvin-share-frontend` in the data directory `/var/lib/pingvin-share`.
+
```

Comment:
This line contains very specific details about the service. The /var/lib directory is standard on NixOS for data and if changes are done to the service in the future, this could be easily missed.